src/client/Microsoft.Identity.Client/Http/IHttpManager.cs

```diff
@@ -41,5 +41,19 @@ internal interface IHttpManager
             StringContent body,
             ILoggerAdapter logger,
             CancellationToken cancellationToken = default);
+
+        Task<HttpResponse> SendPostForceResponseAsync(
```

Comment:
Seems unused?

Reply:
This will be used with IMDS. I added it since I was adding for get.